app/forms/idv/api_document_verification_status_form.rb

```diff
@@ -31,12 +31,17 @@ def remaining_attempts
 
     def timeout_error
       return unless @async_state.timed_out?
-      errors.add(:timeout, t('errors.doc_auth.document_verification_timeout'))
+      errors.add(
+        :timeout, t('errors.doc_auth.document_verification_timeout'),
+        type: :doc_verification
```

Comment:
similar, I think this would make sense as `document_verification_timeout`

```suggestion
        type: :document_verification_timeout
```